app/assets/stylesheets/application.css

```
@@ -22,3 +22,17 @@
 .avatar-icon {
   cursor: pointer;
 }
+
+/* TODO: デザイン　動作確認をしやすいようとりあえず入れた */
+.toast {
+  position: fixed;
+  top: 24px;
+  right: 24px;
+  z-index: 1000;
+
+  padding: 12px 16px;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)